src/RcppExports.cpp

```diff
@@ -178,10 +182,12 @@ static const R_CallMethodDef CallEntries[] = {
     {"_RcppCCTZ_formatDouble", (DL_FUNC) &_RcppCCTZ_formatDouble, 4},
     {"_RcppCCTZ_parseDouble", (DL_FUNC) &_RcppCCTZ_parseDouble, 3},
     {"_RcppCCTZ_now", (DL_FUNC) &_RcppCCTZ_now, 0},
+    {"_RcppCCTZ_getOffset", (DL_FUNC) &_RcppCCTZ_getOffset, 0},
     {NULL, NULL, 0}
 };
 
 RcppExport void R_init_RcppCCTZ(DllInfo *dll) {
+    R_RegisterCCallable("RcppCCTZ", "_RcppCCTZ_getOffset", (DL_FUNC) &_RcppCCTZ_getOffset);
```

Comment:
Registering at the C level for other functions is such a good idea that I am really embarrassed I did not think of it myself.  There is probably scope for one or two more so yes, let's get this started!

Reply:
Yes, will try to think about a coherent set that would make sense. Shall I create an issue to keep track of this?

Reply:
Yes please!  Issues are best for that.